cli/man/splinter-circuit-template-arguments.1.md

```diff
@@ -41,7 +44,9 @@ FLAGS
 ARGUMENTS
 =========
 `TEMPLATE-NAME`
-: Circuit template that contains the arguments of interest.
+: Name of the circuit template containing the arguments of interest. The template
+  name is appended to the default circuit template directory,
+  `/usr/share/splinter/circuit-templates`, to locate the circuit template file.
```

Comment:
Change the second sentence to "The template file must exist in the default circuit template directory, `/usr/share/splinter/circuit-templates`."

Also, if there's no way to change that directory, remove the word "default".

(This comment applies to all man pages with the TEMPLATE-NAME argument.)

Reply:
I changed all of the TEMPLATE-NAME argument descriptions, and removed `default` before `circuit template directory` across all mentions in the man pages that I added